### ISSUE TITLE
Docker base image for pgbouncer

### ISSUE THREAD

**Issue by Harif-Rahman** [pgbouncer/pgbouncer#1020]
Do we have official docker image for pgbouncer?
we are using 1.9.0 version running in opsworks and we wanted to move to eks.
so we are looking for official image which is not running with alpine as base image. since we noticed few issues in alpine.

**Comment by pablogamboa** [comment]
I think most of the people are using bitnami's pgbouncer image, right now on `bitnami/pgbouncer:1.22.0`

**Comment by JelteF** [comment]
Yeah, the bitnami one seems to be the most popular. There is no official image, because I don't have time to maintain that (and build the CI infrastructure to publish it).

**Comment by insspb** [comment]
@JelteF The problem re-raised as bitnami will not support pgbouuncer since August 28
https://github.com/bitnami/containers/issues/83267
